### PR TITLE
Completed JSX Parent Child Assignment

### PR DESCRIPTION
**Pull request by rajesh2355**
# Task Assigned 
image

# Figma Link
[https://www.figma.com/file/Qj3DK3OKab06VQe5l2UbVo/Growth-Capital?node-id=4941-32398&t=DBojEU3KXHcXOS7w-0](https://www.figma.com/file/Qj3DK3OKab06VQe5l2UbVo/Growth-Capital?node-id=4941-32398&t=DBojEU3KXHcXOS7w-0)

# Output Image After Render 
jsx_parent_child

### CODE REVIEW

File: React/jsx_assignment/src/Theme/index.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { createTheme, Theme } from "@mui/material/styles";
+
+export const myTheme: Theme = createTheme({
+  palette: {
+    primary: {
+      // shades of pink
```
Comment (Sahithi-003) [inline React/jsx_assignment/src/Theme/index.tsx:6]: Remove commented out part if not needed!

File: React/jsx_assignment/src/Theme/index.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+      main: "#E39AB2",
+    },
+    secondary: {
+      //shades of black
```
Comment (Sahithi-003) [inline React/jsx_assignment/src/Theme/index.tsx:10]: Remove commented out part if not needed!

File: React/jsx_assignment/src/Theme/index.tsx [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+      light: "#363439",
+      main: "#201F24",
+    },
+    info: {
+      //shades of gray(main) and white(light)
```
Comment (Sahithi-003) [inline React/jsx_assignment/src/Theme/index.tsx:15]: Remove commented out part if not needed!

File: React/jsx_assignment/src/stories/Examples/Button.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+  argTypes: {
+    backgroundColor: { control: 'color' },
+  },
+} as ComponentMeta<typeof Button>;
+
+// More on component templates: https://storybook.js.org/docs/react/writing-stories/introduction#using-args
+const Template: ComponentStory<typeof Button> = (args) => <Button {...args} />;
+
+export const Primary = Template.bind({});
+// More on args: https://storybook.js.org/docs/react/writing-stories/args
```
Comment (Sahithi-003) [inline React/jsx_assignment/src/stories/Examples/Button.stories.tsx:20]: Remove commented code if not needed

**Comment by rajesh2355** [comment]
@Sahithi-003 I have updated the code Please check.

File: React/jsx_assignment/src/assets/Icons/info-circle.svg
Comment (Sahithi-003) [inline React/jsx_assignment/src/assets/Icons/info-circle.svg:1]: Move icons to public folder and you can create atom icon

File: React/jsx_assignment/src/assets/Icons/receipt-item.svg
Comment (Sahithi-003) [inline React/jsx_assignment/src/assets/Icons/receipt-item.svg:1]: Move icons to public folder and you can create atom icon

File: React/jsx_assignment/src/components/Molecules/MidDiv/index.tsx [hunk 5]
```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import ReceiptIcon from "../../Atoms/ReceiptIcon";
+import styled from "@emotion/styled";
+import myTheme from "../../../Theme";
+
+const MidSection = styled.div`
+  height: 80px;
+  width: 80px;
```
Comment (Sahithi-003) [inline React/jsx_assignment/src/components/Molecules/MidDiv/index.tsx:8]: Dont hardcode width and heights concert them into vw and vh

File: React/jsx_assignment/src/components/Organisms/Card/index.tsx [hunk 6]
```diff
@@ -0,0 +1,46 @@
...
+const dateString = "May-03-2023";
+const dateObj = new Date(Date.parse(dateString));
+
+export interface CardProps {
+  dateProp: Date;
+}
+
+const CardDiv = styled.div`
+  width: 350px;
+  height: 260px;
```
Comment (Sahithi-003) [inline React/jsx_assignment/src/components/Organisms/Card/index.tsx:16]: Dont hardcode width and heights concert them into vw and vh

**Review by Sahithi-003 — CHANGES_REQUESTED** [review]
Your folder structure is not correct it should be like this
image
Also please provide pr description appropriately!
Please address review comments!

File: React/jsx_assignment/src/Theme/index.tsx (hunk 1 above)
Comment (rajesh2355) [reply React/jsx_assignment/src/Theme/index.tsx]: Changes done and pushed the updates.

File: React/jsx_assignment/src/Theme/index.tsx (hunk 2 above)
Comment (rajesh2355) [reply React/jsx_assignment/src/Theme/index.tsx]: changes done and pushed the updates.

File: React/jsx_assignment/src/Theme/index.tsx (hunk 3 above)
Comment (rajesh2355) [reply React/jsx_assignment/src/Theme/index.tsx]: updated the code and pushed the changes.

**Review by Sahithi-003 — CHANGES_REQUESTED** [review]
Please keep only files that are to be reviewed and add all remaining to git ignore , Can't go through 68 files please make changes and update the pr

**Comment by rajesh2355** [comment]
@Sahithi-003 I have updated the files. Now this PR only contains the files which have been either added or modified.

**Review by Sahithi-003 — APPROVED** [review]

**Pull request merged**